src/Microsoft.Tye.Hosting/AggregateApplicationProcessor.cs

```diff
@@ -28,10 +29,28 @@ public async Task StartAsync(Application application)
 
         public async Task StopAsync(Application application)
         {
+            var exceptions = new List<Exception>();
```

Comment:
SPICY but...

This is why the composite pattern is the worst. We probably would have already had this code in place if not for the design *feeling simple* when it's not.